### PR TITLE
feat: Blue/green deployment for postgres

### PR DESCRIPTION
**Pull request by magreenbaum**
## Description
Bump provider version to support postgres blue/green deployments. 
Also refactor examples to its own example and remove from complete-mysql since it is not compatible with `manage_master_user_password`. 

## Motivation and Context
https://github.com/hashicorp/terraform-provider-aws/pull/34216

## Breaking Changes
No.

## How Has This Been Tested?
- [x] I have updated at least one of the `examples/*` to demonstrate and validate my change(s)
- [x] I have tested and validated these changes using one or more of the provided `examples/*` projects

- [x] I have executed `pre-commit run -a` on my pull request

### CODE REVIEW

File: examples/blue-green-deployment/README.md [hunk 1]
```diff
@@ -0,0 +1,98 @@
+# Blue/Green deployment example for PostgreSQL
```
Comment (antonbabenko) [inline examples/blue-green-deployment/README.md:1]: Please add link to this example in the main README.md file in the root.

**Review by antonbabenko — APPROVED** [review]
It looks good to me. There is just a small comment.

File: examples/blue-green-deployment/README.md (hunk 1 above)
Comment (magreenbaum) [reply examples/blue-green-deployment/README.md:1]: Updated in https://github.com/terraform-aws-modules/terraform-aws-rds/pull/517/commits/b959874cdf337e3220347a81862e17b34cf64946.

**Comment by antonbabenko** [comment]
This PR is included in version 6.3.0 :tada:

**Pull request merged**